### PR TITLE
Feature/hydrated time entries

### PR DESCRIPTION
**Pull request by khaelys**
- add hydrated time entry dto
- add findAllHydratedTimeEntriesForUserAsync

solves #9

### CODE REVIEW

**Comment by Morasiu** [comment]
I'll look into this on Saturday.(Issues too)

File: Clockify.Net/ClockifyClient.cs [hunk 1]
```diff
@@ -419,6 +419,44 @@ public Task<IRestResponse> DeleteTimeEntryAsync(string workspaceId, string timeE
 
 			return _client.ExecuteGetAsync<List<TimeEntryDtoImpl>>(request);
 		}
```
Comment (Morasiu) [inline Clockify.Net/ClockifyClient.cs:421]: Formating is broken here. It needs some fixes with tabs and brackets.

Comment (khaelys) [reply Clockify.Net/ClockifyClient.cs:421]: I replaced spaces with tabs

File: Clockify.Net/ClockifyClient.cs [hunk 2]
```diff
@@ -386,6 +386,44 @@ public Task<IRestResponse> DeleteTimeEntryAsync(string workspaceId, string timeE
...
+			request.AddQueryParameter(nameof(page), page.ToString());
+			request.AddQueryParameter("page-size", pageSize.ToString());
+
+			return _client.ExecuteGetAsync<List<TimeEntryDtoImpl>>(request);
+		}
+		
+		/// <summary>
+		/// Get hydrated time entries for current user on specified workspace. See Clockify docs for query params explanation.
+		/// </summary>
+		public Task<IRestResponse<List<HydratedTimeEntryDtoImpl>>> FindAllHydratedTimeEntriesForUserAsync(
```
Comment (Morasiu) [inline Clockify.Net/ClockifyClient.cs:426]: Here too :)

**Comment by Morasiu** [comment]
I'll release a new version after this PR 👍

File: Clockify.Net/ClockifyClient.cs (hunk 2 above)
Comment (khaelys) [reply Clockify.Net/ClockifyClient.cs:426]: Sorry, I am having trouble with my IDE, it should be fixed now

**Pull request merged**